src/Pages/Home.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const Home = () => (
+  <div className="home-container">
+    <h1>Welcome to our Page!</h1>
+    <p>
+      Math magicians calculator web application is a convenient tool
+      for performing mathematical calculations.
+      It eliminates the need for a physical calculator, making
+      it accessible from any device with internet access.
+      <br />
+      <br />
+      With this web application, you can perform basic calculations
+      such as addition, subtraction, multiplication,
+      and division as well as more advanced
+      calculations like trigonometry, logarithms, and more.
+    </p>
+
+    <p>
+      One of the most appealing features of this web application is
+      its ability to save and retrieve calculations.
+      This means you can save a calculation and
+      refer to it later, or even share it with others.
+      This feature is especially useful for students, professionals,
+      and anyone who needs to keep track of their calculations for later reference.
+    </p>
```

Comment:
- [ ] [OPTIONAL] Great job with this, @theodoku :wave: :clap: But I think adjusting this width and aligning it at the center might be a little more user-friendly even though styling is not so much a requirement for this implementation. Since the `Navigation` has a padding on both side, It'll be nicer to ensure that all components maintain the padding.

![image](https://user-images.githubusercontent.com/99715640/228456022-cfee5950-a7bc-44d6-bcd3-532225aa7987.png)

- [ ] [OPTIONAL] You can consider reducing the width and aligning it at the center of the page.
- [ ] [OPTIONAL] Kindly consider applying same design to your `Quotes` and the `Let's do some maths` on your `Calculator page. :wink:

Reply:
Implemented changes as you suggested✔